src/main/java/com/aws/greengrass/util/Exec.java

```diff
@@ -87,6 +87,7 @@
     private TimeUnit timeunit = TimeUnit.SECONDS;
     private Copier stderrc;
     private Copier stdoutc;
+    protected long gracefulShutdownTimeout = 5L;
```

Comment:
can this be a duration instead of a unit-less number?

Reply:
Updated to be duration